common/src/main/java/com/microsoft/identity/common/internal/platform/DevicePopManager.java

```diff
@@ -261,17 +264,20 @@
         static final String RSA = "RSA";
     }
 
-    DevicePopManager() throws KeyStoreException, CertificateException,
+    private final Context mContext;
+
+    public DevicePopManager(@NonNull final Context context) throws KeyStoreException, CertificateException,
```

Comment:
I assume this is going to be the android application context rather than an activity context.  One of the safety aspects of passing the context in rather than storing it as a member variable... is that presumably when it's passed in the context is alive and well.  May be worth asking @iambmelt is this was part of his considerations.

Reply:
Yes, this was an intentional design consideration of this API.... I see I'm 5 days too late to give this feedback :(